cbmc_viewer/viewer.py

```diff
@@ -138,9 +140,15 @@ def global_progress(msg, done=False):
 def viewer():
     """Construct the cbmc report."""
 
-    args = create_parser().parse_args()
+    parser = create_parser()
+    args = parser.parse_args()
     args = optionst.defaults(args)
 
+    if not (args.result or args.viewer_result or args.coverage or args.viewer_coverage):
+        print("Nothing to do without property checking or coverage checking results.")
+        parser.print_usage()
+        return 2 # exit with argparse.ArgumentParser.parse_args() "invalid argument" return code
```

Comment:
Sounds like you want [`ArgumentParser.error`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.error)? Even if not, I suggest either using `logging.error` or `print(..., file=sys.stderr)`, I don't think printing to stdout is the right thing here

Reply:
Neat.  Done.